sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobContainerClient.java

```diff
@@ -66,15 +105,64 @@ public final class BlobContainerClient {
     /**
      * Special container name for the logs container in the Storage account.
      */
-    public static final String LOG_CONTAINER_NAME = BlobContainerAsyncClient.LOG_CONTAINER_NAME;
+    public static final String LOG_CONTAINER_NAME = "$logs";
```

Comment:
I'd recommend keeping this pointing to the `LOG_CONTAINER_NAME` in `BlobContainerAsyncClient` so if this were ever updated only one location needs to be updated.